modules/3d/include/opencv2/3d.hpp

```diff
@@ -967,10 +848,12 @@ a 3D point expressed in the world frame into the camera frame:
 CV_EXPORTS_W bool solvePnP( InputArray objectPoints, InputArray imagePoints,
                             InputArray cameraMatrix, InputArray distCoeffs,
                             OutputArray rvec, OutputArray tvec,
-                            bool useExtrinsicGuess = false, int flags = 0 );
+                            bool useExtrinsicGuess = false, int flags = SOLVEPNP_ITERATIVE );
```

Comment:
Any reason to use `0`?

If not, it is clearer for me to use directly the enum flag.

Reply:
It is [changed](https://github.com/opencv/opencv/blame/4d9365990a724654f8cb1b9f3fd9a62e9071b4c8/modules/3d/include/opencv2/3d.hpp#L970) here: #18940
/cc @vpisarev